presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java

```diff
@@ -168,11 +169,18 @@ public void dropTable(ConnectorSession session, ConnectorTableHandle tableHandle
     @Override
     public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorNewTableLayout> layout)
     {
-        JdbcOutputTableHandle handle = jdbcClient.beginCreateTable(tableMetadata);
+        JdbcOutputTableHandle handle = jdbcClient.beginCreateTable(tableMetadata, false);
         setRollback(() -> jdbcClient.rollbackCreateTable(handle));
         return handle;
     }
 
+    @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting)
+    {
+        JdbcOutputTableHandle handle = jdbcClient.beginCreateTable(tableMetadata, ignoreExisting);
+        finishCreateTable(session, handle, emptyList(), emptyList());
```

Comment:
If we fail to finish (rename) the table, we leave a temp table behind.
Am i right?